backend/src/database/sqlite.py

```diff
@@ -72,15 +77,37 @@ def _adapt_list(value: list) -> str:
     return json.dumps(value, separators=(",", ":"))
 
 
+def _adapt_flag(value: BaseFlag) -> str:
+    LOG.debug(f"SQLite._adapt_flag({value=})")
+    return str(value)
+
+
 def _convert_json(value: bytes) -> dict | list:
     return json.loads(value)
 
 
 if sqlite3:
+
     sqlite3.register_adapter(dict, _adapt_dict)
     sqlite3.register_adapter(list, _adapt_list)
+    sqlite3.register_adapter(BaseFlag, _adapt_flag)
     sqlite3.register_converter(SQLITE_JSON_TYPE, _convert_json)
 
+    # Register adapter and converter for all existing Flag subclasses
+    for flag_type in list(BaseFlag.__subclasses__()):
+        LOG.debug(f"Registering adapter and converter for {flag_type=}")
+        sqlite3.register_adapter(flag_type, _adapt_flag)
+
+    # Adapt Flag.__init_subclass__ to register adapter and converter for new Flag subclasses
+    flag_original_init_subclass = BaseFlag.__init_subclass__
+
+    def flag_init_selfregistering_subclass(cls):
+        flag_original_init_subclass()
+        LOG.debug(f"Self-Registering adapter and converter for {cls=}")
+        sqlite3.register_adapter(cls, _adapt_flag)
+
+    BaseFlag.__init_subclass__ = classmethod(flag_init_selfregistering_subclass)
+
```

Comment:
Remove reference to converters in comments and comment-out LOGs
```suggestion
    # Register adapter for all existing Flag subclasses
    for flag_type in list(BaseFlag.__subclasses__()):
        # LOG.debug(f"Registering adapter for {flag_type=}")
        sqlite3.register_adapter(flag_type, _adapt_flag)

    # Adapt Flag.__init_subclass__ to register adapter for new Flag subclasses
    flag_original_init_subclass = BaseFlag.__init_subclass__

    def flag_init_selfregistering_subclass(cls):
        flag_original_init_subclass()
        # LOG.debug(f"Self-Registering adapter for {cls=}")
        sqlite3.register_adapter(cls, _adapt_flag)

    BaseFlag.__init_subclass__ = classmethod(flag_init_selfregistering_subclass)

```